### ISSUE TITLE
Making p3x GitList read only / remove edit buttons?

### ISSUE THREAD

**Issue by Zaptopod4001** [patrikx3/gitlist#32]
Hi,

TL;DR Is it possible to somehow setup this fork of Gitlist to be only used to display repos without editing options? 

I would like to use this as a platform to display my code to some potential employers etc, I really don't want someone be able to modify/sabotage my files... you know what I mean.

I tried your online demo, and to my surprise it was possible to add lines of text to some file with bogus e-mail and user name (although I then removed it). 

This looks pretty nice compared to Gitlist itself which seems to be abandoned, but how can I disable editing?  I'm not very well versed at all with web stuff, but I already got this running on my hosting provider, so this looks otherwise very promising.
